src/components/FormLayout/FormLayout.scss

```diff
@@ -36,4 +41,23 @@ $item-min-size: rem(220px);
     flex-basis: (0.5 * $item-min-size);
     min-width: (0.5 * $item-min-size);
   }
+
+  &.small {
+    flex-basis: #{$modifier-value};
+    max-width: Max(calc(33.33% - #{spacing(loose)}), #{$modifier-value});
```

Comment:
`Max` is taken from [When Sass and New CSS Features Collide](https://css-tricks.com/when-sass-and-new-css-features-collide/#the-solution). Please let me know if there is an alternative to this as using the standard Sass `max` throws an error `...is not a number for max`.